src/Datadog.Trace.ClrProfiler.Native/metadata_builder.cpp

```diff
@@ -24,7 +24,7 @@ HRESULT MetadataBuilder::EmitAssemblyRef(
     assembly_metadata.cbLocale = (DWORD)(assembly_ref.locale.size());
   }
 
-  Info("EmitAssemblyRef ", assembly_ref.str());
+  // Info("EmitAssemblyRef ", assembly_ref.str());
```

Comment:
If it is not useful why not remove it entirely?

Reply:
I was thinking we could restore it when we add a debug mode to the profiler, but I'm going to remove it completely. It's not very helpful. We can always restore it later from history, if needed.